packages/react/src/components/List/ListItem/ListItem.story.jsx

```diff
@@ -46,7 +46,7 @@ export const BasicWKnobs = () => {
           <Button
             key="list-item-edit"
             style={{ color: 'black' }}
-            renderIcon={Edit16}
+            renderIcon={ChartErrorBar16}
```

Comment:
Why this icon change on the ListItem story?

Reply:
Unintentional, and I thought it was being displayed due to the latest `next` not being merged in, but it looks to be perhaps a bad merge somewhere along the way?